### PR TITLE
Connecting /gamemode path into user flow

### PR DESCRIPTION
**Pull request by artofmayhem**
This is a simple PR tying in the new /gamemode logic into user flow. 

The reason for this is to ensure smooth and accurate traversal through the child gameplay path from Parent Dashboard. 

Two locations were changed to accomplish this. Simply pointing the two nodes at each other by changing the path= logic. 

User now flows from Play Game button to Child Dashboard. From the dashboard it pushes to game mode where the user chooses single or multiplayer. 

From there the flow takes user to the reading stage of game play

### CODE REVIEW

File: src/components/pages/Gamemode/Gamemode.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const Gamemode = ({ ...props }) => {
 
   const startSinglePlayerMode = e => {
     e.preventDefault();
-    push('/gameplay');
+    push('/gameplay/mission/read');
```
Comment (Yonathan-Admasu728) [inline src/components/pages/Gamemode/Gamemode.js:49]: great fix here

File: src/components/pages/ChildDashboard/RenderChildDashboard.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const RenderChildDashboard = props => {
   const [modalVisible, setModalVisible] = useState(false);
 
   const handleAcceptMission = e => {
-    push('/gameplay/mission/read');
+    push('/gamemode');
```
Comment (Yonathan-Admasu728) [inline src/components/pages/ChildDashboard/RenderChildDashboard.js:18]: neat

**Review by Yonathan-Admasu728 — APPROVED** [review]

File: src/components/pages/ChildDashboard/RenderChildDashboard.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ const RenderChildDashboard = props => {
   const [modalVisible, setModalVisible] = useState(false);
 
   const handleAcceptMission = e => {
-    push('/gameplay/mission/read');
```
Comment (just-kato) [inline src/components/pages/ChildDashboard/RenderChildDashboard.js:18]: Nice job with removing this path and connected the new one

File: src/components/pages/Gamemode/Gamemode.js (hunk 1 above)
Comment (just-kato) [inline src/components/pages/Gamemode/Gamemode.js:49]: Nice work!

**Review by just-kato — APPROVED** [review]

**Pull request merged**